iOS/Flows/Conversation/Input/ConversationInputAccessoryView.swift

```diff
@@ -165,3 +145,81 @@ class ConversationInputAccessoryView: SwipeableInputAccessoryView {
 }
 
 class AlertProgressView: View {}
+
+extension ConversationInputAccessoryView {
```

Comment:
Moved this into the same file as the class declaration because the file's not that long anymore.